src/Microsoft.Extensions.Logging.Analyzers/LogFormatAnalyzer.cs

```diff
@@ -113,7 +120,10 @@ private void AnalyzeFormatArgument(SyntaxNodeAnalysisContext syntaxContext, Expr
                 if (int.TryParse(valueName, out _))
                 {
                     syntaxContext.ReportDiagnostic(Diagnostic.Create(Descriptors.MEL1NumericsInFormatString, formatExpression.GetLocation()));
-                    break;
+                }
+                else if (char.IsLower(valueName[0]))
```

Comment:
Would it fire on numbers? It's a little strange to show MEL1 and MEL5 at the same place.

Reply:
I'll add a test. I don't believe so though.

Reply:
Actually, the existing tests verify that. They assert that only a single diagnostic is returned for each case.

Reply:
I'll enable MEL0005 on the numeric ones just to ensure it doesn't accidentally happen